.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Bug Report
+about: Let us know what needs fixing
+title: ''
+labels: 'bug :bug:'
+assignees: ''
+
+---
+
+Pre-Report Checklist
+--------------------
+
+- [ ] I am running the latest versions of pyQuil and the Forest SDK
+- [ ] I checked to make sure that this bug has not already been reported
```

Comment:
Does github require you to click these boxes before you can submit? I feel like the types of people who most need to be reminded of these steps are also the least likely to be contentious list-checkers, and for everyone else it's a minor annoyance.

On the other hand duplicate bug reports and reports against pyquil v 0.6 are also an annoyance, so if that's been a problem in practice, so be it.

Reply:
We can definitely drop the checklist if you think it's silly!

Reply:
It just seems that we see a lot of issues with old versions of things, which we'll learn in the context section, so maybe that's enough

Reply:
I don't think it's silly. Maybe people will ignore it. If they do ignore it, we can quickly refer them to the checklist to make sure they're not missing anything trivial.

Reply:
To be clear, I don't think it's silly either.  I'm just ornery when someone makes me fill out a checklist to file a bug report, but that might just be me.

Reply:
I don't think they're mandatory. We won't force you @appleby :)

Reply:
I'm probably also wincing reflexively at the thought of having to click those boxes myself like 50,000 times, lol.

Reply:
> I don't think they're mandatory. We won't force you @appleby :)

*You* won't, but my OCD will :|

Reply:
Theyre not mandatory -- just a friendly reminder :)